content/applications/essentials/contacts.rst

```diff
@@ -0,0 +1,243 @@
+Review` configurations. These are **only** present when the *Enterprise Resellers* module is
+installed.
+
+Membership tab
+--------------
+
+Finally, there is the :guilabel:`Membership` tab, which can help users manage any memberships that
+are being offered to this specific contact. It should be noted that this tab is associated with the
+*Members* app.
+
+Smart buttons
+=============
+
+At the top of the contact form, there are some additional options available, known as *smart
+buttons*.
+
+Here, Odoo displays a variety of records, related to this contact, that were created on other apps.
+Odoo integrates information from every single app, so there are many smart buttons.
+
+.. example::
+   For example, there is an :guilabel:`Opportunities` smart button, where all the opportunities
+   related to this customer from the *CRM* app are accessible.
+
+.. tip::
+   If the corresponding applications are installed, their related smart buttons appear
+   automatically on a contact form.
+
+A user can see any :guilabel:`Meetings`, :guilabel:`Sales`, :guilabel:`POS Orders`,
+:guilabel:`Subscriptions`, project :guilabel:`Tasks`, and the :guilabel:`More` smart button reveals
+additional options, via a drop-down menu. A user can even quickly access :guilabel:`Purchases`,
+:guilabel:`Helpdesk` tasks, :guilabel:`On-time Rate` for deliveries, :guilabel:`Invoiced`
+information, :guilabel:`Vendor Bills`, and the :guilabel:`Partner Ledger` connected to this contact.
+
+Deliveries, documents, loyalty cards, and direct debits are *also* linked to smart buttons, like
+this, should there be any outstanding/on-file for this contact.
+
+If the contact is a partner, the user can visit their partner page on the Odoo-built website by
+clicking the :guilabel:`Go to Website` smart button.
+
+Archive contacts
+----------------
+
+If a user decides they no longer want to have this contact active, the record can be archived. To do
+that, go to the :icon:`fa-cog` :guilabel:`Action` menu at the top of the contact form, and click
+:guilabel:`Archive`.
+
+Then, click :guilabel:`OK` from the resulting :guilabel:`Confirmation` pop-up window.
+
+With this contact successfully archived, as indicated by a banner at the top, they do not show up
+in the main contacts page, but they can still be searched for with the :guilabel:`Archived` filter.
+
+.. tip::
+   A contact can be *unarchived*, if the user decides to work with them again. To do that, just
+   click the :icon:`fa-cog` :guilabel:`Action` menu again at the top of the archived contact form,
+   and click :guilabel:`Unarchive`. Upon doing so, the :guilabel:`Archived` banner is removed, and
+   the contact is restored.
+
+.. seealso::
+   Contact forms can be accessed in the *Sales* application, as well. Go to :menuselection:`Sales
+   app --> Orders --> Customers`, and clear any default filters from the :guilabel:`Search...` bar.
+   Then, click on the desired customer to open their contact form.
+
```

Comment:
This information seems out of place in the seealso, I'd suggest putting it in a tip admonition, or removing it entirely 
```suggestion
```